### ISSUE TITLE
Integrate node-inspector

### ISSUE THREAD

**Issue by knownasilya** [spolu/breach_core#164]
It would be nice to be able to debug modules from dev-tools, maybe [node-inspector](https://github.com/node-inspector/node-inspector) integration is the way to go? Would be super awesome! :neckbeard:

**Comment by spolu** [comment]
In the upcoming releases of Breach, breach_core should be a pure nodeJS app. All available instrumentation in Node will be available to Breach

**Comment by knownasilya** [comment]
@spolu do we have an overarching issue that covers that and references all sub issues?

**Comment by spolu** [comment]
Not really. First step is a major overhaul of exo_browser. Just created that issue: https://github.com/breach/exo_browser/issues/161
